app/src/main/java/org/wikipedia/dataclient/wikidata/Entities.kt

```diff
@@ -23,10 +23,10 @@ class Entities : MwResponse() {
     @Serializable
     class Entity {
         val id: String = ""
-        val labels: Map<String, Label> = emptyMap()
-        val descriptions: Map<String, Label> = emptyMap()
-        val sitelinks: Map<String, SiteLink> = emptyMap()
-        val statements: JsonElement? = null
+        private val labels: JsonElement? = null
```

Comment:
Depending on the context, these fields that come from Wikidata (Wikibase) can actually manifest as old-style collections, where it's an empty array `[]` if there are no items, but a proper object `{...}` if it's nonempty. Therefore these need to be serialized dynamically when accessing. (or we could possibly create a custom deserializer in the future)

If we want to be 100% compatible, this is unfortunately necessary.